doc/howto/benchmark_performance.md

```diff
@@ -14,11 +14,11 @@ If you run this tool repeatedly with different configurations, you can compare t
 
 ## Get the tool
 
-If you’re using the snap, the benchmarking tool is automatically installed.
+If you're using the snap, the benchmarking tool is automatically installed.
 It is available as `lxd.benchmark`.
 
 Otherwise, if you have installed LXD through your distribution's package manager or built from source, the tool should be available as `lxd-benchmark`.
-If it isn't, make sure that you have `go` (version 1.18 or later) installed and install the tool with the following command:
+If it isn't, make sure that you have `go` (version 1.21.1 or later) installed and install the tool with the following command:
```

Comment:
Why 1.21.1 vs just 1.21?

Reply:
github.com/openfga/go-sdk@v0.2.3 requires go@1.21.1

Reply:
This feels like its going to be a maintenance headache. Can we have a link to https://documentation.ubuntu.com/lxd/en/latest/requirements/#go instead of repeating the info in multiple places?